services/web-app/pages/elements/icons/library.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+import { useContext, useEffect } from 'react'
+
+import PageHeader from '@/components/page-header'
+import PageTabs from '@/components/page-tabs'
+import IconLibrary from '@/components/svg-libraries/icon-library'
+import { assetsNavData } from '@/data/nav-data'
+import { LayoutContext } from '@/layouts/layout/layout'
+
+const seo = {
+  title: 'Icons',
+  description:
+    'Icons are visual symbols used to represent ideas, objects, or actions. They ' +
+    'communicate messages at a glance, afford interactivity, and draw attention to important ' +
+    'information.'
+}
+
+const tabs = [
+  {
+    name: 'Library',
+    path: 'library'
+  },
+  {
+    name: 'Usage',
+    path: 'usage'
+  },
+  {
+    name: 'Code',
+    path: 'code'
+  },
+  {
+    name: 'Contribute',
+    path: 'contribute'
+  }
+]
+
+const Library = ({ iconMetadata, iconCategoryMetadata }) => {
+  const { setPrimaryNavData } = useContext(LayoutContext)
+
+  useEffect(() => {
+    setPrimaryNavData(assetsNavData)
+  }, [setPrimaryNavData])
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <PageHeader title={seo.title} withTabs={true} />
+      <PageTabs tabs={tabs} />
+      <IconLibrary iconCategoryMetadata={iconCategoryMetadata} iconMetaData={iconMetadata} />
+    </>
+  )
+}
+
+export const getStaticProps = async () => {
```

Comment:
you could use getStaticProps inside the .mdx file (see services/web-app/pages/designing/get-started.mdx), is there a reason you chose to convert to .js file instead? (just curious here)